<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormEdges</class>
 <widget class="QDialog" name="FormEdges">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>631</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Роли</string>
  </property>
  <widget class="QTableView" name="tvEdges">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>831</width>
     <height>521</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spin_id">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>540</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>999999</number>
   </property>
  </widget>
  <widget class="QComboBox" name="cb_src_private">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>520</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cb_type_private">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>520</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cb_dst_private">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>510</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>600</y>
     <width>831</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="push_add">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Добавить</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="push_save">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Сохранить</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="push_revert">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Отменить</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="push_del">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Удалить</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>540</y>
     <width>781</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Sourse:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QFontComboBox" name="cb_src">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Type:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="cb_type">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Destination:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QFontComboBox" name="cb_dst">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>570</y>
     <width>561</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Capacity:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="spin_capacity"/>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Status:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="spin_status"/>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>isNew:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="spin_isNew"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>570</y>
     <width>261</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Quant:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="cb_quant">
      <property name="enabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QComboBox" name="cb_src11">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>520</y>
     <width>125</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cb_dst11">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>520</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cb_quant_private">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>590</y>
     <width>82</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>tvEdges</zorder>
  <zorder>spin_id</zorder>
  <zorder>cb_src_private</zorder>
  <zorder>cb_type_private</zorder>
  <zorder>cb_dst_private</zorder>
  <zorder>cb_src11</zorder>
  <zorder>cb_dst11</zorder>
  <zorder>cb_quant_private</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
